pkgs/applications/editors/emacs/elisp-packages/yes-no/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchurl, trivialBuild }:
+
+trivialBuild {
+  pname = "yes-no";
+  version = "0";
+
+  src = fetchurl {
+    url = "https://github.com/emacsmirror/emacswiki.org/blob/185fdc34fb1e02b43759ad933d3ee5646b0e78f8/yes-no.el";
+    sha256 = "1k0nn619i82jiqm48k5nk6b8cv2rggh0i5075nhc85a2s9pwhx32";
+  };
+
+  dontUnpack = true;
+
+  installPhase = ''
```

Comment:
Isn't this already done when using `trivialBuild`?

Reply:
Ah... yes, you are right. Thanks!